docs/csharp/discards.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Discards - C# Guide| Microsoft Docs
+description: Describes C#'s support for discards, which are unassigned, discardable variables, and the ways in which discards can be used.
+keywords: .NET,.NET Core
+author: rpetrusha
+ms.author: ronpet
+ms.date: 07/21/2017
+ms.topic: article
+ms.prod: .net
+ms.technology: devlang-csharp
+ms.devlang: csharp
+---
+# Discards - C# Guide
+
+Starting with C# 7, C# supports discards, which are temporary, dummy variables that are intentionally unused in application code. Discards are equivalent to unassigned variables; they do not have a value. Discards can reduce memory allocations and and enhance the readability and maintainability of your code.
+
+You indicate that a variable is a discard by assigning it the underscore (`_`) as its name. For example, the following method call returns a 3-tuple in which the first and second values are discards:
+
+```csharp
+(var -, -, area) = city.GetCityInformation(cityName);
```

Comment:
I don't know if this is the typeface on GitHub, but it looks like these are '-' and not '_'.